src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java

```diff
@@ -43,6 +48,14 @@ public void setType(String type) {
         this.type = type;
     }
 
+    public Boolean getSync() {
+        return sync;
+    }
+
+    public void setSync(Boolean sync) {
```

Comment:
The object is fully constructed after calling the constructor and it should not change (immutability is your friend)